### PR TITLE
fix(69881): cancel item after order --run-build

### PR DESCRIPTION
**Pull request by imam-auliya**
> [#69881] - FE POS  - Done

Acceptance Criteria
- [x] Cancel item after order
- [x] Clear button on cancel item not close the modal
- [x] Differentiate text color based on menu condition(cancel, pre-order, post-order)
- [x] Set default package extra

Changelog:  
- handle cancel item
- re-calculate subtotal

image

image

### CODE REVIEW

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=67) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=67)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=67)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=67)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=67)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=67&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_duplicated_lines_density&view=list&pullRequest=67)

File: src/app/components/Features/Package/Package.component.tsx [hunk 1]
```diff
@@ -179,7 +179,7 @@ const PackageComponent: React.FC<PackageProp> = ({menuItem, changeItem, disabled
                   `Select ${selectedGroup?.max_qty ?? 0} item`}
               </span>
               <br />
-              <span className='text-sm'>{calculateQty()},00 item selected</span>
+              <span className='text-sm'>{calculateQty()} item selected</span>
```
Comment (aldiz1711) [inline src/app/components/Features/Package/Package.component.tsx:182]: image
pastikan ini sesuai dengan figma

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=67) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=67)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=67)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=67)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=67)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=67&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_duplicated_lines_density&view=list&pullRequest=67)

**Review by aldiz1711 — APPROVED** [review]
OK

**Pull request merged**